src/main/java/edu/metrostate/ics499/team2/model/Quiz.java

```diff
@@ -0,0 +1,83 @@
+package edu.metrostate.ics499.team2.model;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "quiz")
+public class Quiz {
```

Comment:
@f947 please perform the following:

1. checkout the main branch (`git checkout main`)
2. pull the main branch (`git pull origin main`)
3. checkout your working branch again (`git checkout feat/abebe`)
4. merge main into your branch (`git merge main`)
5. extend this class (public class Quiz extends Game). Please follow what is implemented in `Flashcard.java` as a reference, but also note not all of what was implemented there applies here as well.
5. Delete the `Games.java` class.
5. Push the changes up (`git push origin feat/abebe`)